check-exercises.hs redundant import warning after GHC 7.10.x

Travis keeps warning with:

```
_test/check-exercises.hs:18:1: warning: [-Wunused-imports]
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()
```

Should check whether that's required. If not required, remove the line completely. If required, add the `()`.

--- a/_test/check-exercises.hs
+++ b/_test/check-exercises.hs
@@ -15,7 +15,10 @@ import System.Process (rawSystem)
 import Data.List (isPrefixOf, intercalate)
 import Data.Maybe (catMaybes)
 import Control.Monad (filterM)
-import Control.Applicative
+
+-- base >= 4.8 re-exports Control.Applicative.<$>.
+import Control.Applicative -- This is only need for <$>,  if GHC <  7.10
+import Prelude             -- This trick avoids a warning if GHC >= 7.10
 
 withTemporaryDirectory_ :: FilePath -> IO a -> IO a
 withTemporaryDirectory_ fp f =

